package.json

```diff
@@ -15,11 +15,15 @@
 		"@typechain/ethers-v5": "^8.0.2",
 		"@types/mocha": "^8.2.2",
 		"@types/node": "^14.14.34",
+		"@types/react-relay": "^11.0.3",
+		"@types/relay-runtime": "^12.0.1",
+		"babel-plugin-relay": "^12.0.0",
 		"lerna": "^3.22.1",
 		"mocha": "^8.4.0",
 		"pinst": "^2.1.6",
 		"ts-node": "^10.0.0",
-		"typescript": "^4.3.2"
+		"typescript": "^4.3.2",
+		"web": "^0.0.2"
```

Comment:
hmm what does this do?

Reply:
That was added by you @tarrencev let me remove it